### ISSUE TITLE
Request:- Custom Rebranding and app name

### ISSUE THREAD

**Issue by Enigma1309** [25huizengek1/ViMusic#35]
### Steps to reproduce the bug

.

### Expected behavior

.

### Actual behavior

.

### ViMusic version

.

### Android version

.

### Additional information

This fork of vi music has came to be the one of the most stable branch, wouldn't it be a good idea to give it a custom name and rebranding?!!
this may include
1. Updated Readme.md (like added features of this branch, like this repo https://github.com/aniyomiorg/aniyomi )
2. New app name
3. New icon
4. Etc...
 
You can achieve those by creating a new discussion about name/icon in GitHub repo
 
This rebranding will give this repo it's Unique essence and attract a wider userbase who are migrating from official repo

**Comment by 25huizengek1** [comment]
Not a bug, so removed that part from the issue. I think it is a good idea (and now that I think of it I maybe have to actually make a 'How to migrate' wiki), however, I do not know how far we can take this license-wise, because I'm not the original maintainer/owner of the project/product. I think as long as we credit the original maintainer/owner (@vfsfitvnm), I think we are OK.

**Comment by 25huizengek1** [comment]
Thank you for your suggestion, will consider. I'm not the best at making decisions but when I do I'll respond to this issue.

**Comment by Enigma1309** [comment]
> I do not know how far we can take this license-wise, because I'm not the original maintainer.
The upstream repo is using GPL 3 license which means the whole repo is copyleft and we have full authority to Rebrand/Share that software
so we are good license and all

>  I'm not the best at making decisions

no one is best but i hope we as a community can bring out the best decision for all of us 

Here are some (unique) names i got from AI suggestions
1. **Cadence**
2. Serenade
3. **Beatify**
4. Harmonic
5. **Voxify**
6. Crescendo

In my opinion **Voxify** and **Cadence** sounds great in general

**Comment by siggi1984** [comment]
> > I do not know how far we can take this license-wise, because I'm not the original maintainer.
> 
> The upstream repo is using GPL 3 license which means the whole repo is copyleft and we have full authority to Rebrand/Share that software so we are good in terms of license and all
> 
> > I'm not the best at making decisions
> 
> no one is best but i hope we as a community can bring out the best decision for all of us
> 
> Here are some (unique) names i got from AI suggestions
> 
> 1. **Cadence**
> 2. Serenade
> 3. **Beatify**
> 4. Harmonic
> 5. **Voxify**
> 6. Crescendo
> 
> In my opinion **Voxify** and **Cadence** sounds great in general

Voxify already exists.

https://play.google.com/store/apps/details?id=com.voxify

**Comment by dtomvan** [comment]
I agree. This repo is way ahead of vfsfitvnm, while he doesn't seem to be active. For the new name, I would keep the "music spirit", though. How about VTunes/ViTunes/VTify/ViTify?

**Comment by devika15** [comment]
Has the original Vimusic been abandoned? No updates for over a year.

**Comment by Enigma1309** [comment]
> Has the original Vimusic been abandoned? No updates for over a year.

Yes it seems to be the case
